hudi-cli/src/main/java/org/apache/hudi/cli/commands/ArchivedCommitsCommand.java

```diff
@@ -225,26 +225,24 @@ private Comparable[] commitDetail(GenericRecord record, String metadataName,
   }
 
   private Comparable[] readCommit(GenericRecord record, boolean skipMetadata) {
-    try {
-      switch (record.get("actionType").toString()) {
-        case HoodieTimeline.CLEAN_ACTION:
-          return commitDetail(record, "hoodieCleanMetadata", skipMetadata);
-        case HoodieTimeline.COMMIT_ACTION:
-        case HoodieTimeline.DELTA_COMMIT_ACTION:
-          return commitDetail(record, "hoodieCommitMetadata", skipMetadata);
-        case HoodieTimeline.ROLLBACK_ACTION:
-          return commitDetail(record, "hoodieRollbackMetadata", skipMetadata);
-        case HoodieTimeline.SAVEPOINT_ACTION:
-          return commitDetail(record, "hoodieSavePointMetadata", skipMetadata);
-        case HoodieTimeline.COMPACTION_ACTION:
-          return commitDetail(record, "hoodieCompactionMetadata", skipMetadata);
-        default: {
-          return new Comparable[] {};
-        }
+    String actionType = record.get("actionType").toString();
+    switch (actionType) {
+      case HoodieTimeline.CLEAN_ACTION:
+        return commitDetail(record, "hoodieCleanMetadata", skipMetadata);
+      case HoodieTimeline.COMMIT_ACTION:
+      case HoodieTimeline.DELTA_COMMIT_ACTION:
+        return commitDetail(record, "hoodieCommitMetadata", skipMetadata);
+      case HoodieTimeline.ROLLBACK_ACTION:
+        return commitDetail(record, "hoodieRollbackMetadata", skipMetadata);
+      case HoodieTimeline.SAVEPOINT_ACTION:
+        return commitDetail(record, "hoodieSavePointMetadata", skipMetadata);
+      case HoodieTimeline.COMPACTION_ACTION:
+        return commitDetail(record, "hoodieCompactionMetadata", skipMetadata);
+      case HoodieTimeline.REPLACE_COMMIT_ACTION:
+        return commitDetail(record, "hoodieReplaceCommitMetadata", skipMetadata);
+      default: {
```

Comment:
how about restore, log compaction, indexing actions?

Reply:
These can be fixed in a separate PR if needed.  Clustering and replace commits are more commonly used than others.